GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -100,6 +102,41 @@ public class Glia {
                     callback()
             }
         }
+
+        var viewFactory = ViewFactory(
+            with: Theme(),
+            messageRenderer: messageRenderer,
+            environment: .init(
+                data: environment.data,
+                uuid: environment.uuid,
+                gcd: environment.gcd,
+                imageViewCache: environment.imageViewCache,
+                timerProviding: environment.timerProviding,
+                uiApplication: environment.uiApplication
+            )
+        )
+
+        callVisualizerCoordinator = .init(
+            viewFactory: viewFactory,
+            presenter: .topViewController(application: .live)
+        )
+        interactor?.addObserver(self) { [weak self] event in
+            guard let engagement = self?.environment.coreSdk.getCurrentEngagement(),
+                  engagement.source == .callVisualizer,
+                  case .screenShareOffer(let answer) = event else {
+                return
+            }
+
+            self?.callVisualizerCoordinator?.offerScreenShare(
+                with: Theme().alertConfiguration.screenShareOffer,
+                accepted: {
+                    // should show bubble here
```

Comment:
Is this a hint for future implementation for bubble?

Reply:
that is removed in next PR
This comment fixed here - 6e1fdab26d89b583c47a68413b7683904268724e